### DISCUSSION TITLE
Unstarring a repo in a list causes it to be re-starred

### DISCUSSION THREAD

**Discussion by MattIPv4** [community/community#8236] [Other Feature Feedback, Questions, & Ideas] [answered]
If you have a starred repo in a list and press the main star button to unstar it, then open up the dropdown again, the repo will be re-starred.

**Comment by bailey** [comment]
Hey Matt! I'm having trouble reproducing this – would you mind sharing a screen capture or screenshots of where this occurs?

**Reply by MattIPv4** [reply]
No worries, here you go:

https://user-images.githubusercontent.com/12371363/144241247-6cedf720-1477-42ec-b593-27d8c34a074d.mov

It seems like the dropdown gets cached in some way, and then makes some API calls when it gets re-opened that causes the repo to be re-starred.

**Reply by bailey** [reply]
Perfect, thanks for sharing this! I'll get it logged with the team right away. ✨

**Comment by bailey** [comment, marked as answer]
This has been fixed 🎉

**Comment by kayzler** [comment]
[DFMiniMp3-master.zip](https://github.com/github/feedback/files/7916580/DFMiniMp3-master.zip)
